.gitignore

```diff
@@ -25,3 +25,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+yarn.lock
```

Comment:
```suggestion
```
this should be pushed in the repo so everyone has same build including production.